src/test/java/com/google/api/codegen/testdata/ruby_doc_main_library.baseline

```diff
@@ -50,133 +54,133 @@ module Library
       attr_reader :stub
 
       # The default address of the service.
-      SERVICE_ADDRESS = 'library-example.googleapis.com'.freeze
+      SERVICE_ADDRESS = "library-example.googleapis.com".freeze
 
       # The default port of the service.
       DEFAULT_SERVICE_PORT = 443
 
-      CODE_GEN_NAME_VERSION = 'gapic/0.1.0'.freeze
+      CODE_GEN_NAME_VERSION = "gapic/0.1.0".freeze
 
       DEFAULT_TIMEOUT = 30
 
       PAGE_DESCRIPTORS = {
-        'list_shelves' => Google::Gax::PageDescriptor.new(
-          'page_token',
-          'next_page_token',
-          'shelves'),
-        'list_books' => Google::Gax::PageDescriptor.new(
-          'page_token',
-          'next_page_token',
-          'books'),
-        'list_strings' => Google::Gax::PageDescriptor.new(
-          'page_token',
-          'next_page_token',
-          'strings')
+        "list_shelves" => Google::Gax::PageDescriptor.new(
+          "page_token",
+          "next_page_token",
+          "shelves"),
+        "list_books" => Google::Gax::PageDescriptor.new(
+          "page_token",
+          "next_page_token",
+          "books"),
+        "list_strings" => Google::Gax::PageDescriptor.new(
+          "page_token",
+          "next_page_token",
+          "strings")
       }.freeze
 
       private_constant :PAGE_DESCRIPTORS
 
       BUNDLE_DESCRIPTORS = {
-        'publish_series' => Google::Gax::BundleDescriptor.new(
-          'books',
+        "publish_series" => Google::Gax::BundleDescriptor.new(
```

Comment:
Actually this is correct. This is confusing, but in Ruby, `{"foo": "bar"}` and `{"foo" => "bar"}` are different. 
- `{"foo" => "bar"}`: normal one, the key is as is seen (in this example, a String instance `"foo"`)
- `{"foo": "bar"}`: special syntax, the key is converted into a Symbol (in this example, `:foo`)

For page descriptors and bundle descriptors, we want to have string-keyed Hash, so this still needs to be `=>`.